components/secret-mgt/org.wso2.carbon.identity.secret.mgt.core/src/main/java/org/wso2/carbon/identity/secret/mgt/core/constant/SQLConstants.java

```diff
@@ -0,0 +1,31 @@
+package org.wso2.carbon.identity.secret.mgt.core.constant;
+
+/**
+ * This class contains database queries related to secret management CRUD operations.
+ */
+public class SQLConstants {
+
+    public static final String INSERT_SECRET = "INSERT INTO IDN_SECRET( ID, TENANT_ID, NAME, VALUE, " +
+            "CREATED_TIME, LAST_MODIFIED ) VALUES(?, ?, ?, ?, ?, ?)";
```

Comment:
let' use column names instead of `?`

Try to change all the occurrence in all SQL queries

Reply:
You can refer here. https://github.com/wso2/carbon-identity-framework/blob/master/components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java#L4617

Reply:
resolved